integration/server/serve.go

```diff
@@ -41,19 +41,21 @@ import (
 )
 
 const (
+	// DefaultNumShards is the default number of shards
+	DefaultNumShards     = 12
```

Comment:
per offline chat, make this a testOption

Reply:
Agreed, that would be ideal.

Reply:
P.S. If it makes it easier you can move everything in the `integration/server` package into just `integration`.  It was already moved from somewhere else that used to actually use it for running the server, so it's not really necessary to be in its own package here.

Reply:
Dunno how much work it would be, but can we make the `integration` server share startup code with `services/m3dbnode`?